tests/plugins/tableselection/manual/draganddropimage.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: dialog, __next__, 547, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tableselection, image, undo, sourcearea, basicstyles
+
+
+1. Open browser's console.
+3. Using drag&drop move image to other table cell.
+
+**Expected**
+
+Image was moved properly, the content looking good and there is no error in the console.
+
+**Unxpected**
+
+Image was not moved.
+There's error in the console.
+After the image was moved, the table was modified - an extra cell was added.
```

Comment:
Please - for `expected` and `unexpected` use notation described in our guide: https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests.

So please split sentences and use list formating for them.